docs/DeveloperGuide.adoc

```diff
@@ -62,7 +71,6 @@ For example, the `Logic` component (see the class diagram given below) defines i
 image::LogicClassDiagram.png[]
```

Comment:
![Screenshot 2019-10-25 at 1 18 27 PM](https://user-images.githubusercontent.com/42567064/67545107-8a26da80-f72a-11e9-8c32-165c531d3762.png)

Words are too small -  not unreadable
May be cluttered with too many unnecessary details
White space can be better used - maximise empty spaces